linode/vpc/framework_schema_resource.go

```diff
@@ -0,0 +1,50 @@
+package vpc
+
+import (
+	"github.com/hashicorp/terraform-plugin-framework/resource/schema"
+	"github.com/hashicorp/terraform-plugin-framework/resource/schema/int64planmodifier"
+	"github.com/hashicorp/terraform-plugin-framework/resource/schema/planmodifier"
+	"github.com/hashicorp/terraform-plugin-framework/resource/schema/stringplanmodifier"
+	"github.com/linode/terraform-provider-linode/linode/helper/customtypes"
+)
+
+var frameworkResourceSchema = schema.Schema{
+	Attributes: map[string]schema.Attribute{
+		"id": schema.Int64Attribute{
+			Description: "The id of the VPC.",
+			Computed:    true,
+			PlanModifiers: []planmodifier.Int64{
+				int64planmodifier.UseStateForUnknown(),
+			},
+		},
+		"label": schema.StringAttribute{
+			Description: "The label of the VPC. Only contains ascii letters, digits and dashes",
+			Required:    true,
+		},
+		"region": schema.StringAttribute{
+			Description: "The region of the VPC.",
+			Required:    true,
```

Comment:
Could a `RequiresReplace` plan modifier be added here since region isn't updatable?

Reply:
Yes, thanks for catching it!